lib/compress/zstd_compress.c

```diff
@@ -137,11 +137,12 @@ ZSTD_CCtx* ZSTD_initStaticCCtx(void* workspace, size_t workspaceSize)
     ZSTD_cwksp_move(&cctx->workspace, &ws);
     cctx->staticSize = workspaceSize;
 
-    /* statically sized space. entropyWorkspace never moves (but prev/next block swap places) */
+    /* statically sized space. tmpWorkspace never moves (but prev/next block swap places) */
     if (!ZSTD_cwksp_check_available(&cctx->workspace, ENTROPY_WORKSPACE_SIZE + 2 * sizeof(ZSTD_compressedBlockState_t))) return NULL;
```

Comment:
Don't we basically have to replace every usage of `ENTROPY_WORKSPACE_SIZE` with `TMP_WORKSPACE_SIZE`? Including here?

Reply:
Not every instance, since in some cases it's really "just" about entropy,
but quite possibly here.
This might be a left over.